contributing/running-server-side-experiment.md

```diff
@@ -0,0 +1,128 @@
+<!---
+Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Running Experiment with Different Binaries
+
+We understand that experimenting a new feature or change on the client side is not always practial. For example, if you rewrite a big chunk of the code base, or include add extra step to optimize code in the transpiling process, you want to run the experiment at the server level, where different bianries will be returned.
```

Comment:
include add extra step to optimize code in the transpiling process -> or include a new transformer in the transpiling process, you would want to run...